interface/therapy_groups/therapy_groups_views/addGroup.php

```diff
@@ -0,0 +1,167 @@
+                                <?php foreach($users as $user): ?>
+                                    <option value="<?php echo attr($user['id']);?>" <?php echo !is_null($groupData['counselors']) && in_array($user['id'], $groupData['counselors']) ? 'selected' : '';?>><?php echo text($user['fname'] . ' ' . $user['lname']);?></option>
+                                <?php endforeach; ?>
+                            </select>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-6">
+                    <div class="row">
+                        <div class="col-md-3 col-sm-5">
+                            <span class="bold"><?php echo xlt('Notes'); ?>:</span>
+                        </div>
+                        <div class="col-md-9 col-sm-7">
+                            <textarea name="group_notes" class="full-width" style="height: 70px"><?php echo text($groupData['group_notes']);?></textarea>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row group-row">
+                <div class="col-md-6">
+                    <div class="row">
+                        <div class="col-md-4 col-sm-5">
+                            <span class="bold"><?php echo xlt('Guest counselors'); ?>:</span>
+                        </div>
+                        <div class="col-md-8 col-sm-7">
+                           <input type="text" name="group_guest_counselors" class="full-width"  value="<?php echo attr($groupData['group_guest_counselors']);?>">
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row group-row">
+                <div class="col-md-3">
+                    <button type="submit" name="save" value="save" <?php echo $savingStatus == 'success' ? 'disabled' : '';?>><?php echo xlt('Add group');?></button>
+                </div>
+                <div class="col-md-9 col-sm 12">
+
+                    <?php if($savingStatus == 'exist'): ?>
+                        <div id="exist-group"><h4 class="group-error-msg"><?php echo text($message) ?></h4><button id="cancel-save"><?php echo xlt('cancel') ?></button><button type="submit" value="save_anyway" name="save"><?php echo xlt('Creating anyway') ?></button></div>
+                    <?php endif ?>
+                    <?php if($savingStatus == 'success'): ?>
+                        <h4 class="group-success-msg"><?php echo text($message) ?></h4>
+                    <?php endif ?>
+                    <?php if($savingStatus == 'failed'): ?>
+                        <h4 class="group-serror-msg"><?php echo text($message) ?></h4>
+                    <?php endif ?>
+                </div>
+            <div>
+        </form>
+    </div>
+</main>
+<script>
+    $(document).ready(function(){
+       $('.datepicker').datepicker({
```

Comment:
Added translation using the openemr translation methods. Left some more options for text translations in comment so can be used if needed.